src/Ionide.ProjInfo/Library.fs

```diff
@@ -792,27 +800,29 @@ type WorkspaceLoaderViaProjectGraph private (toolsPath, ?globalProperties: (stri
 
                 bm.EndBuild()
 
-                let buildProjs =
-                    result.ResultsByNode.Keys
-                    |> Seq.collect (fun (pgn: ProjectGraphNode) -> seq { yield pgn.ProjectInstance })
-                    |> Seq.toList
 
                 logger.info (
                     Log.setMessage "{overallCode}, projects built {count} {projects} "
-                    >> Log.addContextDestructured "count" (buildProjs |> Seq.length)
+                    // >> Log.addContextDestructured "count" (buildProjs |> Seq.length)
                     >> Log.addContextDestructured "overallCode" result.OverallResult
                     >> Log.addExn result.Exception
                 )
 
+
+                let buildProjs =
+                    result.ResultsByNode.Keys
+                    |> Seq.collect (fun (pgn: ProjectGraphNode) -> seq { yield pgn.ProjectInstance })
+                    |> Seq.toList
+
```

Comment:
Had to move this because this could fail before we log the error from the design time build.  Could add another log for how many we parsed.